package.json

```diff
@@ -7,10 +7,10 @@
   "module": "build/esm/index.js",
   "exports": {
     "node": {
-      "module": "./build/esm/index.js",
-      "import": "./build/main/index.js"
+      "import": "./build/esm/index.js",
+      "require": "./build/main/index.js"
     },
-    "default": "./build/main/index.js"
+    "default": "./build/esm/index.js"
```

Comment:
Non-blocking: For consistency, shouldn't `default`'s value be equal to `main`'s even if `exports` takes precedence?

Reply:
I think this is the issue I am encountering when using the package in node:

```
/usr/bin/node /home/fryorcraken/.nvm/versions/node/v16.14.0/lib/node_modules/yarn/bin/yarn.js run predefined-test
(node:219199) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)

/home/fryorcraken/src/status-im/waku-tests/node_modules/@waku/staging/build/esm/index.js:1
export { generatePrivateKey, generateSymmetricKey, getPublicKey, } from "./lib/crypto";
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1032:15)
    at Module._compile (node:internal/modules/cjs/loader:1067:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:168:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:337:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:437:15)
    at async formattedImport (/home/fryorcraken/src/status-im/waku-tests/node_modules/mocha/lib/nodejs/esm-utils.js:7:14)
    at async Object.exports.requireOrImport (/home/fryorcraken/src/status-im/waku-tests/node_modules/mocha/lib/nodejs/esm-utils.js:48:32)
    at async Object.exports.loadFilesAsync (/home/fryorcraken/src/status-im/waku-tests/node_modules/mocha/lib/nodejs/esm-utils.js:103:20)
    at async singleRun (/home/fryorcraken/src/status-im/waku-tests/node_modules/mocha/lib/cli/run-helpers.js:125:3)
    at async Object.exports.handler (/home/fryorcraken/src/status-im/waku-tests/node_modules/mocha/lib/cli/run.js:374:5)

Process finished with exit code 1

```

https://github.com/status-im/waku-tests/compare/master...test-exports-js-waku